src/util/notify.hpp

```diff
@@ -29,8 +29,9 @@ namespace big::notify
 			if (g.reactions.crash.log)
 				LOG(WARNING) << "Blocked " << crash << " crash from " << player->get_name() << " ("
 				             << (player->get_net_data() ? player->get_net_data()->m_gamer_handle.m_rockstar_id : 0) << ")";
-		
-			g.reactions.crash.process_common(g_player_service->get_by_id(player->m_player_id));
+			
+			g.reactions.crash.process(g_player_service->get_by_id(player->m_player_id), true);
+			g.reactions.crash.process_common(g_player_service->get_by_id(player->m_player_id), false);
```

Comment:
Ok this is weird, this feels a lot like a bandage fix, if this can be sufficiently backed up on why it was implemented like this I could allow it but for now it seems like it would be a good idea to do a partial rewrite.